src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -155,13 +169,21 @@ private async Task<IEnumerable<IAccount>> GetAccountsInternalAsync(ApiIds apiId,
             return cacheAndBrokerAccounts;
         }
 
-        private async Task<IEnumerable<IAccount>> GetAccountsFromBrokerAsync(string homeAccountIdFilter)
+        private async Task<IEnumerable<IAccount>> GetAccountsFromBrokerAsync(
+            string homeAccountIdFilter,
+            ICacheSessionManager cacheSessionManager,
+            RequestContext requestContext)
         {
             if (AppConfig.IsBrokerEnabled && ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth())
             {
                 var broker = ServiceBundle.PlatformProxy.CreateBroker(ServiceBundle.Config, null);
                 var brokerAccounts =
-                    (await broker.GetAccountsAsync(AppConfig.ClientId, AppConfig.RedirectUri).ConfigureAwait(false))
+                    (await broker.GetAccountsAsync(
```

Comment:
The broker interface is much more complex now, as it needs to be able to fetch accounts from the cache, save a token response etc.